custom/icds_reports/ucr/data_sources/dashboard/child_tasks.json

```diff
@@ -184,7 +184,15 @@
           "6g_vit_a_6",
           "6g_vit_a_7",
           "6g_vit_a_8",
-          "7g_vit_a_9"
+          "7g_vit_a_9",
+          "anc_1",
```

Comment:
This is good for now as these new fields are not used anywhere. But generally its good idea to put the ucr changes in a groundwork PR

Reply:
okay